src/algorithms/PVT/libs/rtcm_printer.cc

```diff
@@ -225,12 +225,9 @@ Rtcm_Printer::~Rtcm_Printer()
 void Rtcm_Printer::Print_Rtcm_Messages(const Rtklib_Solver* pvt_solver,
     const std::map<int, Gnss_Synchro>& gnss_observables_map,
     double rx_time,
-    int32_t type_of_rx,
-    int32_t rtcm_MSM_rate_ms,
-    int32_t rtcm_MT1019_rate_ms,
-    int32_t rtcm_MT1020_rate_ms,
-    int32_t rtcm_MT1045_rate_ms,
+    uint32_t signal_enabled_flags,
     int32_t rtcm_MT1077_rate_ms,
+    int32_t rtcm_MT1087_rate_ms,
     int32_t rtcm_MT1097_rate_ms,
     bool flag_write_RTCM_MSM_output,
     bool flag_write_RTCM_1019_output,
```

Comment:
We can only use the flag variable and don't need the rates here, because if you look in `rtklib_pvt_gs.cc` we already do this:
```
if (d_rtcm_MT1019_rate_ms != 0)  // allows deactivating messages by setting rate = 0
    {
        if (current_RX_time_ms % d_rtcm_MT1019_rate_ms == 0)
            {
                flag_write_RTCM_1019_output = true;
            }
    }
```

Where I am confused, is that `rtcm_MSM_rate_ms` is the same as `rtcm_MT1097_rate_ms`, but we allow to change `rtcm_MT1077_rate_ms` and `rtcm_MT1087_rate_ms` in the config, and there is no "flag" to tell the printer it's time to log. I feel we need to add a flag for these two rates, but I wanted to get feedback before doing this change.